### ISSUE TITLE
hard coded WORST_POSSIBLE_RESULT

### ISSUE THREAD

**Issue by tsinggggg** [automl/auto-sklearn#771]
WORST_POSSIBLE_RESULT for training loss is hard coded to be 1.0. It makes sense for some evaluation metrics but not all, for example when MSE is used and a ’Crashed' trial would get a training loss of 1. it'd be great if the worst result could depend on the metric.

**Comment by mfeurer** [comment]
Thank you very much for reporting this issue @tsinggggg. We have fixed it in #845. Please reopen this issue if this fix does not work for you.
